app/shared/modals/reservation-cancel/__tests__/ReservationCancelModalContainer.test.js

```diff
@@ -32,123 +30,14 @@ describe('shared/modals/reservation-cancel/ReservationCancelModalContainer', ()
   }
 
   describe('render', () => {
-    test('renders a Modal component', () => {
-      const modalComponent = getWrapper().find(Modal);
+    test('renders a ReservationCancelModal component', () => {
+      const modalComponent = getWrapper().find(ReservationCancelModal);
       expect(modalComponent.length).toBe(1);
     });
 
-    describe('Modal header', () => {
```

Comment:
Would it make sense to move these to `ReservationCancelModal.test.js` instead of deleting?

Reply:
Yes, I moved all the relevant unit tests to `ReservationCancelModal.test.js`.